src/ui/buttons/buttons.jsx

```diff
@@ -110,7 +110,7 @@ export function Buttons(props : ButtonsProps) : ElementNode {
     }
 
     if (fundingSources.indexOf(FUNDING.CARD) !== -1) {
-        fundingSources = fundingSources.filter(src => src !== FUNDING.CARD).concat([ FUNDING.CARD ]);
+        fundingSources = [ ...fundingSources.filter(src => src !== FUNDING.CARD),  FUNDING.CARD ];
```

Comment:
should we ignore this to prevent the prod change?

Reply:
what rule is this

Reply:
I'll look into it. I didn't change any rules in grumbler-scripts so I don't know off hand but should be easy to find. I guessing maybe the prefers-spread from unicorn is set to something. thats off the top of my head

Reply:
you know what i just want to ignore this too for now the more i think about it so gonna lint ignore here as well.